pyproject.toml

```diff
@@ -7,3 +7,59 @@ extend-exclude = '''
   | devenv
 )/
 '''
+
+[tool.vendoring]
+destination = "conda/_vendor/"
+requirements = "conda/_vendor/vendor.txt"
+namespace = "conda._vendor"
+
+protected-files = ["__init__.py", "vendor.txt", "README.rst"]
+patches-dir = "tools/vendoring/patches"
+
+[tool.vendoring.transformations]
+substitute = []
+drop = [
+  # contains unnecessary scripts
+  "bin/",
+  # remove unwanted modules
+  '^boltons/(cache|debug|depr|dict|easter|eco|exc|file|format|func|gc|io|iter|json|list|math|mbox|named|path|queue|socket|stats|str|table|tb||type|url)utils\.py',
```

Comment:
Interestingly boltons is available on conda-forge, maybe it's something to look into relying for conda as a proper dependency.

Reply:
Yup but I think this is out of scope for this PR.